fh_immuta_utils/client.py

```diff
@@ -103,6 +103,10 @@ def make_glob_request_headers(cls, config: Dict[str, Any]) -> Dict[str, Any]:
             return cls.make_generic_odbc_request_headers(config)
         if config["handler_type"] == "Amazon Athena":
             return cls.make_athena_glob_request_headers(config)
+        if config["handler_type"] == "Snowflake":
+            headers = cls.make_generic_odbc_request_headers(config)
+            headers["sql-warehouse"] = config["warehouse"]
+            return headers
```

Comment:
it's a bit strange that codecov is calling these lines out as not being covered isn't it? I see tests added, is it possible this code path is still not being reached by the tests?

Reply:
Correct, it is not being reached. Historically I'm not sure why tests don't exist. `client.py` is made up mostly of the functions that make the API requests. This class instantiates the client. We'll have to get some mocking going here to test this.

Reply:
Clarifying that once I'm back, I'll add the test coverage for this.

Reply:
Tests added!

Reply:
lgtm!